### ISSUE TITLE
Mullvad Does Not Work on FreeBSD and OpenBSD

### ISSUE THREAD

**Issue by markmcb** [tailscale/tailscale#12099]
### What is the issue?

The tailscale client can be setup on both FreeBSD and OpenBSD with Mullvad enabled from the web settings and accepts the --exit-node option for tailscale up.

The problem is It's clear the client is not technically capable of working on these operating systems with Mullvad as an exit node (and probably any exit node). I would expect both the web setting and CLI client to inform the user this feature is not implemented rather than silently failing.

### Steps to reproduce

Setup FreeBSD or OpenBSD tailscale client with Mullvad exit node.

### OS

Other

### OS version

FreeBSD 14.0-Release & OpenBSD 7.5

### Tailscale version

1.66.0 (FreeBSD) and 1.60.1 (OpenBSD)

### Bug report

BUG-26677d069aff08efeb309ab77aca8d937770763166d40f821a6c8171fb562634-20240511021635Z-38c0c7dd060d64cf

**Comment by larihuttunen** [comment]
I guess this issue is still unresolved?

**Comment by bradfitz** [comment]
I agree that we should have a nicer error message at least in the short term.

The tracking bug for exit node support on BSDs is #10024 which depends on #17321

**Comment by markmcb** [comment]
This is working now for me on OpenBSD. I edited the title and report to only reflect FreeBSD.
